### PR TITLE
Remove JSBins from routing/specifying-a-routes-model

### PR DESCRIPTION
**Pull request by mamut**
Extracted snippets that weren't already on the page.

Here an ember-twiddle for the second one: http://ember-twiddle.com/56bd90ed50442bfc51ef

Meta PR: https://github.com/emberjs/guides/issues/483

### CODE REVIEW

File: source/routing/specifying-a-routes-model.md [hunk 1]
```diff
@@ -204,7 +202,31 @@ will always execute the model hook.
 If your data represented by your model is being updated frequently, you may
 want to refresh it periodically:
 
-<a class="jsbin-embed" href="http://jsbin.com/sefuv/2/embed?js">JS Bin</a><script src="http://static.jsbin.com/js/embed.js"></script>
+```app/routes/pull-requests.js
+export default Ember.Route.extend({
+  model() {
+    var url = 'https://api.github.com/repos/emberjs/ember.js/pulls';
+    return Ember.$.getJSON(url).then(function(data) {
```
Comment (locks) [inline source/routing/specifying-a-routes-model.md]: is there an `ic-ajax` alternative?

Comment (michaelrkn) [inline source/routing/specifying-a-routes-model.md]: what do you have mind @locks? Also the point of this PR is just to get rid of the jsbin - I think changing around the examples on this page is out of scope, although may warrant a new issue.

Comment (mamut) [inline source/routing/specifying-a-routes-model.md]: `Ember.$.getJSON` is used in other examples on this page – I agree with @michaelrkn that it warrants a new PR.

Sorry for late response, quite busy this week.

Comment (michaelrkn) [inline source/routing/specifying-a-routes-model.md]: Okay, see https://github.com/emberjs/guides/issues/542.
